tests/cart_items.js

```diff
@@ -1,86 +1,142 @@
+import checkoutOverviewPage from "../Pages/checkoutOverviewPage.js";
+
+const validUserName = 'standard_user';
+const validPassword = 'secret_sauce';
+const baseUrl = 'https://www.saucedemo.com/';
+const randomIndex = Math.floor(Math.random() * 6);
+const productDetailsPageUrl = 'https://www.saucedemo.com/inventory-item.html?id=';
+const myFirstName = 'Mia';
+const myLastName = 'J';
+const myZipCode = '407280';
 
 
 fixture`Add product to cart`
     .page(baseUrl)
 
 
-test(`Add one item to cart`, async t => {
-    await t
-        .typeText(userNameInput, validUserName)
-        .typeText(passwordInput, validPassword)
-        .click(loginButton)
-        .click(firstProductAddToCart)
-
-    const cartNumber = await shoppingCartCount.innerText;
-
-    await t.expect(cartNumber).eql('1');
+test(`Add first item to cart`, async t => {
+    await loginPage.typeUserName(validUserName);
+    await loginPage.typePassword(validPassword);
+    await loginPage.clickOnLoginBttn();
+    await productsPage.addProductToCart(0);
+    await t.expect(productsPage.cartCount.innerText).eql('1');
+    await t.expect(shoppingCartPage.cartListProdNo.exists).eql(false);
 });
 
 
 test(`Remove product from cart`, async t => {
-    await t
-        .typeText(userNameInput, validUserName)
-        .typeText(passwordInput, validPassword)
-        .click(loginButton)
-        .click(firstProductAddToCart)
-        .click(shoppingCartButton)
-        .click(firstProductRemoveFromCart)
-
-    const isCartItemVisible = await cartItem.visible;
-    const isProductInCart = await shoppingCartCount.exists;
-
-    await t.expect(isCartItemVisible).eql(false)
-    await t.expect(isProductInCart).eql(false)
+    await loginPage.typeUserName(validUserName);
+    await loginPage.typePassword(validPassword);
+    await loginPage.clickOnLoginBttn();
+    await productsPage.addProductToCart(0);
+    await productsPage.goToCart();
+    await shoppingCartPage.removeSauceBkpFromCart();
+    await t.expect(productsPage.cartCount.visible).eql(false);
 });
 
 
 test(`Add two products to cart and remove one`, async t => {
-    await t 
-         .typeText(userNameInput, validUserName)
-         .typeText(passwordInput, validPassword)
-         .click(loginButton)
-         .click(firstProductAddToCart)
-         .click(secondProductAddToCart)
-         .click(shoppingCartButton)
-         .click(secondProductRemoveFromCart)
-
-    const cartNumber = await shoppingCartCount.innerText;
-    const cartListProductNo = await cartQuantity.innerText;
-
-    await t.expect(cartNumber).eql('1');
-    await t.expect(cartListProductNo).eql('1'); 
+    await loginPage.typeUserName(validUserName);
+    await loginPage.typePassword(validPassword);
+    await loginPage.clickOnLoginBttn();
+    await productsPage.addProductToCart(0);
+    await productsPage.addProductToCart(0);
+    await productsPage.goToCart();
+    await shoppingCartPage.removeSauceBkpFromCart();
```

Comment:
This test needs to be updated, because it uses a function that doesn't exist anymore.